### PR TITLE
[chore #164692058]add the supporting files and connecting to database

### PR DESCRIPTION
**Pull request by ViaChristopheH**
#### What does this PR do?
* Adds the database connecting file
* Connects to the database

#### Description of Task to be completed?
* The database is connected to the queries

#### How should this be manually tested?
* Find the `package.json` file and run the script entitled `dbEngine`
* Run the script using `npm run dbEngine`

#### Any background context you want to provide?
* None 

#### What are the relevant pivotal tracker stories?
#164692058

#### Screenshots (if appropriate)
* None

#### Questions:
* None

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,8 @@
-AUTHENTICATION_TOKEN: 
-JSONWT_SECRET='secret12345678'
\ No newline at end of file
+JWTPRIVATEKEY='anything@epicMAIL'
```
Comment (fob413) [inline .env]: This file shouldn't be online. They contain sensitive information which should not be open for people to see/have access.

File: server/database/dbTables.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+const databaseTables = () => {
+
+    const usersTable = `
+    CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(25) NOT NULL UNIQUE,
+    password VARCHAR(250) NOT NULL,
+    first_name  VARCHAR(25) NOT NULL,
+    last_name VARCHAR(25) NOT NULL,
+    id_group INT
```
Comment (fob413) [inline server/database/dbTables.js]: What is this particular column for?

**Review by fob413 — CHANGES_REQUESTED** [review]
Please address the comments in your PR

File: .env (hunk 1 above)
Comment (ViaChristopheH) [reply .env]: Already addressed

File: server/database/dbTables.js (hunk 2 above)
Comment (ViaChristopheH) [reply server/database/dbTables.js]: It was found unnecessary and then removed.

**Review by fob413 — APPROVED** [review]
Go ahead and merge

**Pull request merged**